pypesto/store/read_from_hdf5.py

```diff
@@ -331,3 +339,45 @@ def read_result(filename: str,
                            f'within {filename}.')
 
     return result
+    
+    
+def read_ensemble_prediction_from_h5(predictor: Callable[[Sequence], PredictionResult],
+                                     input_file: str,
+                                     base_path: str = None):
+    # ToDo read bounds
+    base = Path('')
+    if base_path is not None:
+        base = Path(base_path)
+
+    # open file
+    f = h5py.File(input_file, 'r')
+
+    prediction_id = str(f[os.path.join(base, PREDICTION_ID)][()], 'utf-8')
+
+    pred_res_list = []
+    for i_pred in range(5): #len(f.keys())-1):
```

Comment:
need to replace range(5). Probably easier to just loop over the keys of `f` and skip `prediction_id`?